website/docs/update-all.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
can we add this as part of quick start instead of a new page? we might need to refactor a bit

Reply:
mention both options and users can choose